deploy-web/src/utils/deploymentData/v1beta3.ts

```diff
@@ -1,107 +1,30 @@
-import { CustomValidationError, DeploymentGroups, Manifest, ManifestVersion, getCurrentHeight, getSdl, parseSizeStr } from "./helpers";
+import { CustomValidationError, DeploymentGroups, getCurrentHeight, getSdl, Manifest, ManifestVersion, parseSizeStr } from "./helpers";
 import { defaultInitialDeposit } from "../constants";
 import { stringToBoolean } from "../stringUtils";
 import path from "path";
 import yaml from "js-yaml";
-import { getUsdcDenom } from "@src/hooks/useDenom";
-import { SDL } from "@akashnetwork/akashjs/build/sdl";
+import { getSelectedNetwork } from "@src/hooks/useSelectedNetwork";
+import { NetworkId } from "@akashnetwork/akashjs/build/types/network";
 
 export const endpointNameValidationRegex = /^[a-z]+[-_\da-z]+$/;
-export const endpointKindIP = "ip";
 
-function validate(yamlStr: string, yamlJson) {
+function validate(yamlStr: string, yamlJson, networkId: NetworkId) {
   try {
-    SDL.validate(yamlStr);
+    // TODO: use result of this in client code instead of just using validation
```

Comment:
It's only called at one place in `NewDeploymentData`, maybe we can replace it there.

Reply:
Actually, `getSdl` is called multiple times `DeploymentGroups` `Manifest` `ManifestVersion`, perhaps there's a better to handle this.

Reply:
yes, definitely. I've just left it for the upcoming changes. When I move out all the validation it would be easier to streamline code duplication